frontend/src/components/DefaultInput/input.tsx

```diff
@@ -0,0 +1,16 @@
+import { InputHTMLAttributes } from "react";
+import * as S from "./styles";
+
+type InputProps = {
+} & InputHTMLAttributes<HTMLInputElement>
+
+
+export const Input = ({ name = '', type = 'text', ...rest}: InputProps) => {
+  return ( 
+    <>
+        <S.Container>
+            <S.Input type={type} name={name} {...rest} />
```

Comment:
Em continuação ao caso das `props` lá em cima, aqui vc já passa todas as props de uma vez:

```suggestion
            <S.Input {...props} />
```